councilmatic/settings.py

```diff
@@ -12,29 +12,75 @@
 
 # Build paths inside the project like this: os.path.join(BASE_DIR, ...)
 import os
-from .settings_deployment import *
+
+import dj_database_url
+
 from .settings_jurisdiction import *
 
 
 BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
 
-
 # Quick-start development settings - unsuitable for production
 # See https://docs.djangoproject.com/en/1.8/howto/deployment/checklist/
+allowed_hosts = os.getenv('DJANGO_ALLOWED_HOSTS', [])
+ALLOWED_HOSTS = allowed_hosts.split(',') if allowed_hosts else [
+    'localhost',
+    '127.0.0.1',
+    '0.0.0.0',
+]
 
+DEBUG = False if os.getenv('DJANGO_DEBUG', True) == 'False' else True
 
-try:
-    from .settings_deployment import ALLOWED_HOSTS
-except ImportError:
-    ALLOWED_HOSTS = [
-        'localhost', '127.0.0.1',
-        '.datamade.us',
-        '.councilmatic.org'
-    ]
+if DEBUG:
+    # Set INTERNAL_IPS to use django-debug-toolbar in debug mode
+    import socket
+    hostname, _, ips = socket.gethostbyname_ex(socket.gethostname())
+    INTERNAL_IPS = [ip[:-1] + '1' for ip in ips] + ['127.0.0.1', '10.0.2.2']
 
+SHOW_TEST_EVENTS = False if os.getenv('DJANGO_SHOW_TEST_EVENTS', True) == 'False' else True
+
+STATIC_ROOT = os.getenv('DJANGO_STATIC_ROOT', os.path.join(BASE_DIR, 'static'))
+
+# Nullable keys
+ANALYTICS_TRACKING_CODE = os.getenv('DJANGO_ANALYTICS_TRACKING_CODE', '')
+GOOGLE_API_KEY = os.getenv('DJANGO_GOOGLE_API_KEY', '')
+AWS_KEY = os.getenv('AWS_KEY', '')
+AWS_SECRET = os.getenv('AWS_SECRET', '')
+
+if os.getenv('DJANGO_STATICFILES_STORAGE', False):
+    STATICFILES_STORAGE = os.environ['DJANGO_STATICFILES_STORAGE']
 
-# Application definition
 
+# Database and Solr connections
+DATABASES = {}
+
+DATABASES['default'] = dj_database_url.parse(
+    os.getenv('DATABASE_URL', 'postgis://postgres@postgres:5432/lametro'),
+    conn_max_age=600,
+    ssl_require=True if os.getenv('POSTGRES_REQUIRE_SSL') else False
+)
+DATABASES['default']['ENGINE'] = 'django.contrib.gis.db.backends.postgis'
```

Comment:
Needed to set this manually. The Heroku `DATABASE_URL` always uses the plain Postgres engine, regardless of whether you've installed PostGIS.